dts/bindings/pwm/sifli,sf32lb-pwm.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2025, Qingsong Gou <gouqs@hotmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Sifli PWM controller
+
+compatible: "sifli,sf32lb-pwm"
```

Comment:
Sifli doesn't have a single PWM controller; the implementation will depend on the  timer used. This will mean multiple drivers, one for each timer, so better use here "sifli,sf32lb-gpt-pwm"